src/TrueOrFalse.Frontend.Web/Views/ApiVueApp/Components/Comment/CommentAddController.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace VueApp;
+
+public class CommentAddController(
+    SessionUser _sessionUser,
+    CommentRepository _commentRepository,
+    UserReadingRepo _userReadingRepo,
+    IHttpContextAccessor _httpContextAccessor) : Controller
+{
+    [AccessOnlyAsLoggedIn]
+    [HttpPost]
+    public bool SaveComment([FromBody] AddCommentJson json)
+    {
+        SaveComment(CommentType.AnswerQuestion, json, _sessionUser.UserId);
+        return true;
+    }
+
+    private void SaveComment(CommentType type, AddCommentJson json, int userId)
+    {
+        var comment = new Comment();
+        comment.Type = type;
+        comment.TypeId = json.id;
+        comment.Text = json.text;
+        comment.Title = json.title;
+        comment.Creator = _userReadingRepo.GetById(userId);
+
+        _commentRepository.Create(comment);
+    }
+
+    public record struct SaveAnswerResult(int Id, 
+        string CreatorName, 
+        string CreationDate,
+        string CreationDateNiceText,
+        string CreatorImgUrl,
+        string Title,
+        string Text,
+        bool ShouldBeImproved,
+        bool ShouldBeDeleted,
+        List<string> ShouldReasons,
+        bool IsSettled,
+        List<CommentModel> Answers,
+        int AnswersSettledCount,
+        bool ShowSettledAnswers,
+        string CreatorUrl); 
+
+    [AccessOnlyAsLoggedIn]
+    [HttpPost]
+    public SaveAnswerResult? SaveAnswer([FromBody]AddAnswerType addAnswerType)
+    {
+        var parentComment = _commentRepository.GetById(addAnswerType.commentId);
+
+        if (parentComment.IsSettled)
+        {
+            return null;
```

Comment:
does the frontend handle null returns? snackbar/alert/message on settled discussions